src/homework2/task2.py

```diff
@@ -14,10 +14,19 @@ def longest_word(str_):
     """
 
     # write your code here
-    return ''  # write return value here
+    import string
+    punct = string.punctuation + ' '
+    for el in punct:
```

Comment:
O(n*m) solution. n - number of marks, m - length of the string. 
can be made with O(m)

Reply:
done